script/update-git.js

```diff
@@ -58,10 +64,14 @@ async function getLatestStableRelease() {
   const sortedTags = semver.sort(releaseTags)
   const latestTag = sortedTags[sortedTags.length - 1]
 
-  return latestTag
+  return latestTag.toString()
```

Comment:
This is inferred as a `string|SemVer` object, so forcing it to a `string` here ensures we're working with the right shape.